jsk_perception/CMakeLists.txt

```diff
@@ -229,6 +233,9 @@ if(OPENMP_FOUND)
   jsk_add_nodelet(src/saliency_map_generator_node.cpp "jsk_perception/SaliencyMapGenerator" "saliency_map_generator")
   jsk_add_nodelet(src/skeletonization_nodelet.cpp "jsk_perception/Skeletonization" "skeletonization")
 endif()
+if("$ENV{ROS_DISTRO}" STRGREATER "hydro" AND EXISTS ${libcmt_INCLUDE_DIRS}/libcmt/CMT.h) # >= indigo and has header file
```

Comment:
fixed version of libcmt is not released in stable repo, we have to remove this once sync has done.